.travis.yml

```diff
@@ -10,6 +9,6 @@ branches:
     - master
 
 script:
-  - npm run lint
   - npm run build
   - npm run umd
+  - npm run lint
```

Comment:
Needs to be last so that `native.js` which is now linted doesn't fail for import errors for `./lib` references.